### DISCUSSION TITLE
模型合并转换基础模型选择

### DISCUSSION THREAD

**Discussion by afpro** [ymcui/Chinese-LLaMA-Alpaca-2#41] [Q&A] [answered]
可以把Lora合并到 `meta-llama/Llama-2-7b-chat-hf` 吗

**Comment by airaria** [comment, marked as answer]
chinese-llama2-lora和chinese-alpaca2-lora只适合合并到非chat版本的llama上，即Llama-2-7b-hf

**Reply by azyu11** [reply]
请问对chinese-alpaca2-1.3b进行增量预训练后得到的lora应该和原版哪个模型进行合并呢，也是Llama-2-7b-hf吗
